portfolio/pom.xml

```diff
@@ -29,6 +29,12 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
```

Comment:
nit - keep these dependencies alphabetical by groupId...

Reply:
Ok I'll adjust that.